paddle/framework/error.h

```diff
@@ -122,15 +117,13 @@ class Error {
    */
   bool isOK() const { return msg_ == nullptr; }
 
-  /**
-   * @brief check this status by glog.
-   * @note It is a temp method used during cleaning Paddle code. It will be
-   *       removed later.
-   */
-  void check() const { CHECK(this->isOK()) << msg(); }
-
-private:
+ private:
   std::shared_ptr<std::string> msg_;
```

Comment:
Well, `Error` object creation will do one memory operation. But after that, we'll pass error object using reference all the time, we can also use `Error` as return types of functions as we have [return value optimization](https://en.wikipedia.org/wiki/Return_value_optimization)